recipes/pyviz/meta.yaml

```
@@ -0,0 +1,70 @@
+{% set name = "pyviz" %}
+{% set version = "0.10.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pyviz/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: bcf261c81b0aa5b17f05557519dae0eccdb57c6632ab8aa1c54c958e7c8435b7
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
```

```suggestion
  entry_points:
    - pyviz = pyviz.__main__:main
    
```

Missing entry point needed for Windows...